uswds/uswds-site
# USWDS-Site: Add known issues to component pages

# Summary

- Added known issues to component pages.
- Updated the data keys to use snake case instead of camel case

The known issues were listed in the [Moving from usability test findings to issues/actions](https://docs.google.com/document/d/1f4ZiCkqU-qB-utPrkaLP7i_Gu3nhernCtHKZEsfV3C0/edit) document. (Google doc :lock:)

> [!Note]
> Build issues fixed in 0d4cbe5d.

## Related issue

Closes uswds/uswds-site#2375

## Preview link

- [Character count page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/character-count/)
- [Combo box page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/combo-box/)
- [Date picker page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/date-picker/)
- [File input page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/file-input/)
- [Input mask page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/input-mask/)
- [Validation page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-known-issues/components/validation/)

## Problem statement
Add the known issues listed in the [Moving from usability test findings to issues/actions](https://docs.google.com/document/d/1f4ZiCkqU-qB-utPrkaLP7i_Gu3nhernCtHKZEsfV3C0/edit) document. (Google doc :lock:)  

## Testing and review

1. All issues in the [Moving from usability test findings to issues/actions](https://docs.google.com/document/d/1f4ZiCkqU-qB-utPrkaLP7i_Gu3nhernCtHKZEsfV3C0/edit) doc are included in the known issues section of component pages. 
1. Alert appears as expected on the top of each page that has known issues.
2. The link in the alert works.
3. In the "All known issues for [component name] are tracked in GitHub." copy:
    1. Confirm that the GitHub link points to a list of issues tagged with the appropriate "package" label.
    2. Confirm that the "[component name]" is correct for the page.
4. Issue summaries are accurate, easy to understand, and follow consistent format.
5. Issue links work as expected.
6. Changelogs exist and are free from error.

## Testing checklist

- [ ] All components have changelog entries
- [ ] Known issues are appearing in Character count, Combo box, Date picker, File input, Input mask, and Validation
